src/main/java/com/jagrosh/jmusicbot/audio/AudioHandler.java

```diff
@@ -59,12 +60,28 @@ public class AudioHandler extends AudioEventAdapter implements AudioSendHandler
     private final long guildId;
     
     private AudioFrame lastFrame;
+    private AbstractQueue<QueuedTrack> queue;
 
     protected AudioHandler(PlayerManager manager, Guild guild, AudioPlayer player)
     {
         this.manager = manager;
         this.audioPlayer = player;
         this.guildId = guild.getIdLong();
+        this.queue = new FairQueue<>();
```

Comment:
Can this be removed, or will Java think that the queue might be left unset & doesn't recognize the function call below?

Reply:
The method `setQueueType()` requires the queue to already be initialized, since it calls the "copy constructors" which copy the internal queue of the currently active queue.
I do not know a more elegant solution off the top of my head, any ideas?

Reply:
Adding a null-check to the `AbstractQueue` constructor works, I will do that for now